<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDialog</class>
 <widget class="QDialog" name="ChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat with AlDawry</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1E2633; /* Dark background matching MainWindow */
color: #E0E6ED; /* Light gray text */</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>600</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="headerLabel">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #252A38; /* Slightly lighter header */
border-bottom: 2px solid #00FF7F; /* Green accent */
padding: 10px;
color: #E0E6ED;</string>
      </property>
      <property name="text">
       <string>Chat with AlDawry AI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="chatDisplay">
      <property name="styleSheet">
       <string notr="true">background-color: #252A38; /* Slightly lighter than main bg */
border: 1px solid #2A3A4A; /* Subtle border */
border-radius: 5px;
padding: 10px;
color: #E0E6ED;
font-family: 'Segoe UI', Arial, sans-serif;
font-size: 12px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI','Arial','sans-serif'; font-size:12px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-weight:700;&quot;&gt;AlDawry:&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI';&quot;&gt; Hello! How can I assist you today?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="inputWidget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #1E2633; /* Matches dialog bg */
border-top: 1px solid #2A3A4A; /* Subtle separator */
padding: 10px;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="messageInput">
         <property name="styleSheet">
          <string notr="true">background-color: #2A3240; /* Dark input field */
border: 1px solid #3A4A5A; /* Subtle border */
border-radius: 5px;
padding: 5px;
color: #E0E6ED;
font-family: 'Segoe UI', Arial, sans-serif;
font-size: 12px;</string>
         </property>
         <property name="placeholderText">
          <string>Type your message...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="styleSheet">
          <string notr="true">background-color: #00FF7F; /* Green accent */
color: #1A2633; /* Dark text for contrast */
font-size: 12px;
font-weight: bold;
border-radius: 5px;
padding: 6px 12px;</string>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
